packages/tocco-ui/src/EditableValue/EditableValue.stories.js

```diff
@@ -56,118 +58,151 @@ storiesOf('EditableValue', module)
+        ChildComponent={TextEdit}
+        readOnly={boolean('readOnly', false) || undefined}
+        value={'This is the first line.\nTo be continued...'}
+      />
+  ).add(
     'Url',
-    () => <EditableValueStory ChildComponent={UrlEdit} value={'http://www.tocco.ch'} />
-  )
-  .add(
+    () =>
+      <EditableValueStory
+        ChildComponent={UrlEdit}
+        readOnly={boolean('readOnly', false) || undefined}
+        value={'http://www.tocco.ch'}
+      />
+  ).add(
     'Phone',
-    () => <EditableValueStory ChildComponent={PhoneEdit} value={'+41761234567'} />
-  )
-  .add(
+    () =>
+      <EditableValueStory
+        ChildComponent={PhoneEdit}
+        readOnly={boolean('readOnly', false) || undefined}
+        value={'+41761234567'}
+      />
+  ).add(
     'Number',
-    () => <EditableValueStory
-      ChildComponent={NumberEdit}
-      value={12345}
-      options={{prePointDigits: number('prePointDigits', 8), postPointDigits: number('postPointDigits', 2)}}
-    />
-  )
-  .add(
+    () =>
+      <EditableValueStory
+        ChildComponent={NumberEdit}
+        options={{prePointDigits: number('prePointDigits', 8), postPointDigits: number('postPointDigits', 2)}}
+        readOnly={boolean('readOnly', false) || undefined}
+        value={12345}
+      />
+  ).add(
     'Decimal',
-    () => <EditableValueStory
-      ChildComponent={NumberEdit}
-      value={54321}
-      options={{prePointDigits: number('prePointDigits', 10), postPointDigits: number('postPointDigits', 3)}}
-    />
-  )
-  .add(
+    () =>
+      <EditableValueStory
+        ChildComponent={NumberEdit}
+        options={{prePointDigits: number('prePointDigits', 10), postPointDigits: number('postPointDigits', 3)}}
+        readOnly={boolean('readOnly', false) || undefined}
+        value={54321}
+      />
+  ).add(
     'Integer',
-    () => <EditableValueStory
-      ChildComponent={IntegerEdit}
-      value={200}
-      options={{minValue: number('minValue', 0), maxValue: number('maxValue', 300)}}
-    />
-  )
-  .add(
+    () =>
+      <EditableValueStory
+        ChildComponent={IntegerEdit}
+        options={{minValue: number('minValue', 0), maxValue: number('maxValue', 300)}}
+        readOnly={boolean('readOnly', false) || undefined}
+        value={200}
+      />
+  ).add(
     'MoneyAmount',
-    () => <EditableValueStory
-      ChildComponent={MoneyEdit}
-      value={1234.56}
-      options={{prePointDigits: number('prePointDigits', 8), postPointDigits: number('postPointDigits', 2)}}
-    />
-  )
-  .add(
+    () =>
+      <EditableValueStory
+        ChildComponent={MoneyEdit}
+        options={{prePointDigits: number('prePointDigits', 8), postPointDigits: number('postPointDigits', 2)}}
+        readOnly={boolean('readOnly', false) || undefined}
+        value={1234.56}
+      />
+  ).add(
     'Single-Select',
-    () => <SelectStory
-      isMulti={false}
-      delay={0}
-    />
-  )
-  .add(
+    () =>
+      <SelectStory
```

Comment:
`<SelectStory />` needs `readOnly` set to its `<Select />` child component too. Right now readOnly on all SelectStories don't work.